src/shadowbox/scripts/update_mmdb.sh

```diff
@@ -22,7 +22,7 @@ for monthdelta in $(seq 10); do
 done
 
 gunzip "$TMPDIR/$FILENAME.gz"
-LIBDIR="/var/lib/libmaxminddb"
+DB_DIR="/root/shadowbox/persisted-state"
```

Comment:
I'm not sure what you mean by this. The state dir is mounted from the hosts's state dir, so when we write it into the container state dir it should also then exist in the host.

This means that shadowbox containers start with potentially out of date db's but that problem already exists. We could fix it by running update_mmdb.sh in the cmd instead of in the Dockerfile.

Reply:
The overlay doesn't work like that.
You may write the IP DB into /root/shadowbox/persisted-state during image creation. But on docker run that directory will be replaced with the directory in the host, which will not have the IP DB.

We could, maybe, run update_mmdb on startup (in cmd.sh), but that means that if there are issues with the fetching of the IP DB, the server won't have the db available. We've had issues multiple times in the past. It will also block the startup (which may not be a big deal, since it seems to be fast)

I'd say it's helpful to have an IP DB in the image, as a backup in case of issues. But it would have to be placed somewhere other than the `persisted-state` so it doesn't get replaced.

Reply:
Understood.  For now let's not do anything with `persisted-state` to try to keep this PR as focused as possible.  I'll follow up with a PR which figures out our issue with the db